app/models/api/Address.scala

```diff
@@ -26,9 +26,10 @@ object Country {
 }
 
 case class Address(line1: String,
-                   line2: String,
+                   line2: Option[String] = None,
```

Comment:
You'll need to update the block builders to reflect the optionality of this field so we don't end up sending Null values to DES.

Reply:
Updated